src/USER-OMP/reaxc_forces_omp.cpp

```diff
@@ -289,9 +289,10 @@ void Validate_ListsOMP( reax_system *system, storage * /*workspace */, reax_list
       else comp = bonds->num_intrs;
 
       if (End_Index(i, bonds) > comp) {
-        fprintf( stderr, "step%d-bondchk failed: i=%d end(i)=%d str(i+1)=%d\n",
-                 step, i, End_Index(i,bonds), comp );
-        MPI_Abort( comm, INSUFFICIENT_MEMORY );
+        char errmsg[256];
+        snprintf(errmsg, 256, "step%d-bondchk failed: i=%d end(i)=%d str(i+1)=%d\n",
+                  step, i, End_Index(i,bonds), comp );
+        system->error_ptr->all(FLERR,errmsg);
```

Comment:
Are you sure this code triggers for all MPI ranks at the same time? otherwise, you have to use `error->one()`.